tests/urlpatterns/path_urls.py

```diff
@@ -11,5 +11,7 @@
     path('users/', views.empty_view, name='users'),
     path('users/<id>/', views.empty_view, name='user-with-id'),
     path('included_urls/', include('urlpatterns_reverse.included_urls')),
+    re_path(r'^regex/(?P<pk>[0-9]+)/$', views.empty_view, name='regex'),
+    path('', include('urlpatterns_reverse.extra_urls')),
```

Comment:
It would be better to use or create urls in `tests/urlpatterns` rather than add a dependency on another app.